### PR TITLE
Add Pandas

### PR DESCRIPTION
**Pull request by rgaiacs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,23 @@ In alphabetic order.
     </td>
   </tr>
 
+
+  <tr>
+    <td>
+      <img width="300px" src="" />
```
Comment (rgaiacs) [inline README.md]: Need image here.

File: organization/team.md [hunk 2]
```diff
@@ -50,6 +50,12 @@ organizing Google Summer of Code for NumFOCUS.
 | Phil Elson         |                 |                |
 | Thomas Caswell     |                 |                |
 
+#### Pandas
+
+| Name                | Melange Profile | GitHub Profile |
+|---------------------|-----------------|----------------|
+| Jeff Reback         |                 |                |
```
Comment (rgaiacs) [inline organization/team.md]: More mentors.

Comment (jreback) [inline organization/team.md]: Stephan Hoyer (@shoyer)
me (@jreback)

Comment (jreback) [inline organization/team.md]: will add some more soon

Comment (rgaiacs) [inline organization/team.md]: @shoyer and @jreback How many organizations you are affiliate for GSoC? Because I can't invite you to NumFOCUS. :-)

Comment (jreback) [inline organization/team.md]: not sure what that means?

Comment (jreback) [inline organization/team.md]: @rgaiacs 

pls also add: https://github.com/DevasenaInupakutika as a mentor, specically for a IO on R datasets

Comment (rgaiacs) [inline organization/team.md]: Done. @jreback Could you let me know if someone didn't receive the email invitation?

**Pull request merged**